### DISCUSSION TITLE
bigfix/11.0/webui/WebUI/Admin_Guide/c_log_locations

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#86] [Q&A]
# bigfix/11.0/webui/WebUI/Admin_Guide/c_log_locations

All WebUI logs are stored in one default location. Logs are stored on the WebUI Server in the following locations (unless changed by the server setting _WebUI_Logging_LogPath).

https://help.hcl-software.com/bigfix/11.0/webui/WebUI/Admin_Guide/c_log_locations.html

**Comment by cjcastro** [comment]
Hello,

I didn't change any log locations using the parameters described in the link above.
If the WebUI is installed remotely the path for the service-wrapper log file is at:
C:\Program Files (x86)\BigFix Enterprise\BES WebUI

The service-wrapper log is in a separate directory, the one above.
Also the logs directory is not capitalized, just lowercase. Not sure about Linux, it may be lowercase too. This makes a difference in Linux as incorrect capitalization will lead to an error when looking for a file.

As for the other logs, they are at:
C:\Program Files (x86)\BigFix Enterprise\BES WebUI\WebUI\logs


I think over the years the path for the WebUI might have changed
from: 
C:\Program Files (x86)\BigFix Enterprise\WebUI

to:
C:\Program Files (x86)\BigFix Enterprise\BES WebUI\WebUI\


I'm running a standalone WebUI install on a BigFix server 11.0.2 installation.

Can you please check if the on this page is still relevant, else update it?


Thank you,

Christian
